### PR TITLE
Add id and for attributes to tooltip and alter styling

### PR DESCRIPTION
**Pull request by SEAjamieD**
Adjust so that tooltips render
Chef Automate (10)

### CODE REVIEW

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html [hunk 1]
```diff
@@ -56,17 +56,18 @@
               <chef-td (click)="filterFor('manager_type', node.manager)">{{ node.manager }}</chef-td>
               <chef-td (click)="filterFor('last_contact', node.last_contact)">{{ node.last_contact }}</chef-td>
               <chef-td>
-                <chef-icon class="status-icon" [ngClass]="node.scan_data.status"
+                <chef-icon [attr.id]="node.name" class="status-icon--pointer" [ngClass]="node.scan_data.status"
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html:59]: in order for chef-tooltips to show, the "target" or thing you hover over needs to have an `id` that matches the `for` attribute on the chef-tooltip.

Comment (SEAjamieD) [reply components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html:59]: i also added a small pointer utility class that you can choose to do away with if you want - just giving the cursor a pointer when hovering the icon

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss [hunk 2]
```diff
@@ -25,13 +25,17 @@ chef-icon.SKIPPED {
...
+  }
+}
+
 .run-info-tooltip {
   border-left: $spacing-03 solid;
   border-right: $spacing-03 solid transparent;
   padding-right: $spacing-03;
-  bottom: 100%;
-  left: 50%;
-  margin-left: -60px;
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss:34]: tooltip placement is generated by the `chef-tooltip` component, so adding our own placement on top of that will throw it off.

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss [hunk 3]
```diff
@@ -44,10 +48,6 @@ chef-icon.SKIPPED {
   &.UKNOWN, &.SKIPPED {
     border-left-color: $status-unknown;
   }
-  &.hydrated { 
-    visibility: visible;
-    opacity: 1;
-  }
```
Comment (SEAjamieD) [inline components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss:50]: no need to manually hydrate.  I haven't seen an instance so far where this was needed manually but I don't know much about it.

**Comment by SEAjamieD** [comment]
I've only added to the first Icon of each set in Last Scans

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss (hunk 3 above)
Comment (vjeffrey) [reply components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss:50]: ya this was me trying hard to force it to show up

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss (hunk 2 above)
Comment (vjeffrey) [reply components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.scss:34]: that makes sense!

File: components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html (hunk 1 above)
Comment (vjeffrey) [reply components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html:59]: thank you!! that's the 🔑 -- thank you so much!

do we have a place where our components are documented - like in the code or anything? i didn't actually take the time to look (i should have) but wondering for my future self.

Comment (SEAjamieD) [reply components/automate-ui/src/app/modules/nodes/nodes-list/nodes-list.component.html:59]: cd into components/chef-ui-library
npm install
npm start
navigate to localhost:3333

There is pretty solid documentation for each component there!

**Pull request merged**